desserts/1-medium/chocolate-cupcakes.md

```diff
@@ -0,0 +1,55 @@
+# Chocolate cupcakes
+
+This recipe is adapted from a vegetarian cookbook, and it's quickly become a favorite for its delicious frosting and fluffy base!
+
+**Prep time:** 1 hour
+
+**Serves** 12 cupcakes
+
+## Ingredients
+
+**For the cupcake**
+* 175g plain or all-purpose flour
+* 2tsp cocoa powder
+* 1.5tsp baking powder
+* 140g sugar
+* 25g ground almonds
+* 2 large eggs
+* 175g full-fat yogurt
+* 2-3 drops vanilla extract
+* 25g butter, melted
+* 2tsp rapeseed oil
+
+**For the frosting**
+* 25g dark chocolate, finely chopped
+* 1tbsp dark chocolate, grated for decorating
+* 25g butter, softened
+* 50g icing (powdered) sugar
+* 1tbsp cocoa powder
+* 1tsp semi-skimmed milk
+* 100g low-fat curd cheese
+* 100g quark or mascarpone cheese
+
+### Notes
+I used higher-fat milk and cheeses for the frosting, which still led to good results.
+
+For the dry ingredients of the cupcake, I recommend sifting.
+
+I found that this recipe yields more frosting than is needed, but it also seems good as a spread if you do have some left over.
+
+## Method
+
+1. Preheat the oven to 180 degrees celsius and prepare your baking tray, lining it with baking paper.
+2. In a large bowl, add the flour and the cocoa powder.
+3. Add the baking powder, sugar and almonds, then make a 'hole' in the center of the bowl.
+4. In another bowl, beat the eggs and combine them with the yogurt and vanilla essence.
+5. Add the egg mix to the center of the large bowl, then mix everything for a short period of time, just until the mixture is homogenous.
```

Comment:
I appreciate the use of the word homogenous, but maybe 'until it all comes together'? That might be a too idiomatic use of English on my part, however.